recipes/nanodbc/all/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.14)
```

```suggestion
cmake_minimum_required(VERSION 3.4)
```

Check comment by @danimtb  https://github.com/conan-io/conan-center-index/pull/2460/files#r470514373